tests/test_utils_data_cleaning_critics.py

```diff
@@ -116,15 +114,16 @@ def test_validate_critics(self):
             }
 
         input_df =pd.DataFrame(df_to_test)
-        cleaner._validate(input_df)
 
-        flag = True
-        self.assertTrue(flag)
+        try:
+            cleaner._validate(input_df) # pylint: disable=W0212
+        except Exception: # pylint: disable=W0703
```

Comment:
I think it would be better to take this try/except out entirely, but another way you can do this is 
```
try:
    stuff
except Exception as e
    raise e
```
To get the original traceback. 
But seeing the traceback is the point of tests, so why hide them with a try?